lib/routes/sis001/author.ts

```diff
@@ -22,11 +23,17 @@ export const route: Route = {
     handler,
 };
 
-async function handler(ctx) {
+async function handler(ctx: Context) {
     const { id = '13131575' } = ctx.req.param();
+    let baseUrl = ctx.req.query('baseUrl');
+    if (!baseUrl) {
+        baseUrl = defaultBaseUrl;
+    }
```

Comment:
This brings GHSA-3p3p-cgj7-vgw3 and GHSA-64wp-jh9p-5cg2 again

Reply:
Moved to env config.